src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -79,6 +79,10 @@
       window.EnderecoIntegrator.config.ux.allowCloseModal = !!('{{ page.endereco_config.enderecoAllowCloseIcon }}');
       window.EnderecoIntegrator.config.ux.confirmWithCheckbox = !!('{{ page.endereco_config.enderecoConfirmWithCheckbox }}');
       window.EnderecoIntegrator.config.ux.changeFieldsOrder = true;
+      window.EnderecoIntegrator.config.ux.showPhoneErrors = {{ page.endereco_config.enderecoShowPhoneErrors ? 'true':'false' }};
```

Comment:
Here I would like to have an additional setting called `window.EnderecoIntegrator.config.ux.showPhoneFlag = true;` to enable the display of the country flags.

Reply:
Done :+1:

Reply:
GJ